sources/packages/web/src/services/http/StudentAppealApi.ts

```diff
@@ -14,4 +19,23 @@ export class StudentAppealApi extends HttpBaseClient {
       studentAppeal,
     );
   }
+
+  async getStudentAppealWithRequests(
+    appealId: number,
+  ): Promise<StudentAppealApiOutDTO> {
+    return this.getCallTyped<StudentAppealApiOutDTO>(
+      this.addClientRoot(`appeal/${appealId}/requests`),
+    );
+  }
+
+  async approveStudentAppealRequests(
+    appealId: number,
+    approvals: StudentAppealRequestApiInDTO[],
+  ): Promise<void> {
+    const payload = { requests: approvals } as StudentAppealApprovalApiInDTO;
```

Comment:
can skip using **as** here.

Reply:
We can, it will work, but it was added to enphasisys the typing.
![image](https://user-images.githubusercontent.com/61259237/162051692-de979528-a26a-446e-a5f4-66151cd3c724.png)
vs
![image](https://user-images.githubusercontent.com/61259237/162051874-5518d6a4-ac04-4131-8da1-2a3a53a42c97.png)

Reply:
But as we discussed in our last sync up, using as has a disadvantage that it will not validate the object if all mandatory fields are supplied. So we decided to skip using as unless it is required while assigning entity objects like {id:userId} as User

Reply:
The discussion was specifically for the context of **returning a DTO from API**. I do not believe that we would avoid casting everywhere. It is a Team understanding? @ann-aot @guru-aot @henryoforeh-aot 
There are cases where it would not even be optional, we need them, as below.
![image](https://user-images.githubusercontent.com/61259237/162072999-6c69f197-cf9c-47ff-a572-1b5e06a55c32.png)

Reply:
yes, what @andrewsignori-aot mentioned was also my understanding as well

Reply:
Talked with @dheepak-aot about it on a call and we will probably extend the conversation on dev sync up later.
Removing it for now.

Reply:
The code will be changed to be like the below. Typing the patchCall will make the type explicit also.
![image](https://user-images.githubusercontent.com/61259237/162079623-f96389df-fa68-4b2a-9596-3e612e43065d.png)